developer-docs-site/docs/nodes/node-files.md

```diff
@@ -80,3 +98,12 @@ Fullnode means either a validator fullnode or a public fullnode.
   ```bash
   wget -O fullnode.yaml https://raw.githubusercontent.com/aptos-labs/aptos-core/mainnet/docker/compose/aptos-node/fullnode.yaml
   ```
+
+## haproxy-fullnode.cfg (fullnode only)
+
+- **Git repo:** `aptos-core`
+- **Git branch:** `main` on https://github.com/aptos-labs/aptos-core
+- **Command to download:**
+  ```bash
```

Comment:
The branches are inconsistent here, sometimes mainnet, sometimes mainnet deployment sometimes main

Lets make sure

1) all files present on mainnet_deploytment
2) all the references here are mainnet_deployment if this is for mainnet

Is this document only for mainnet?

Reply:
I was following your note in this Slack thread: https://aptos-org.slack.com/archives/C03EG004E56/p1665531016071549?thread_ts=1665529285.939749&cid=C03EG004E56
but I wasn't sure if I understood what you were saying. Hence my request to review this aspect. 

Documents will now be by default for mainnet. I will mention wherever testnet, devnet can be used. But for now, all mainnet.